midend/expandLookahead.cpp

```diff
@@ -41,16 +41,16 @@ void DoExpandLookahead::expandSetValid(const IR::Expression* base, const IR::Typ
 const IR::Expression* DoExpandLookahead::expand(
     const IR::PathExpression* base, const IR::Type* type, unsigned* offset) {
     if (type->is<IR::Type_Struct>() || type->is<IR::Type_Header>()) {
-        auto vec = new IR::Vector<IR::Expression>();
+        auto vec = new IR::ListExpression({});
```

Comment:
Most of The code that creates an empty ListExpression and then pushes stuff into it should be changed to call LestExpression{ ... } with the elements of the list directly as an initializer list expression.  Since an empty ListExpression should (never? rarely?) happen, the empty ctor should be unnecessary.